src/features/operator/Market/MarketWithdraw/index.tsx

```diff
@@ -133,125 +133,59 @@ export const MarketWithdraw = (): JSX.Element => {
+
+    let currencyAmount = Big(1);
     if (isLbtcAssetId(assetId, network)) {
-      amountInFiatOrLBTC = formatLbtcUnitToSats(amount.toNumber(), lbtcUnit);
-      amountInFiatOrLBTC = fromSatsToUnitOrFractional(
-        Number(amountInFiatOrLBTC),
-        assetPrecision,
-        true,
-        'L-BTC'
+      currencyAmount = Big(
+        fromSatsToUnitOrFractional(
+          Number(formatLbtcUnitToSats(assetAmount.toNumber(), lbtcUnit)),
+          assetPrecision,
+          true,
+          'L-BTC'
+        )
       );
     } else {
-      amountInFiatOrLBTC = amount.toString();
+      currencyAmount = assetAmount;
     }
 
     let rateMultiplier = 1;
-    let preferredCurrencyAmount = Big(1);
     if (assetTicker === LBTC_TICKER[network]) {
       rateMultiplier = prices?.[LBTC_COINGECKOID]?.[currency.value] || 1;
-      preferredCurrencyAmount = Big(amountInFiatOrLBTC).times(rateMultiplier);
+      currencyAmount = Big(currencyAmount).times(rateMultiplier);
     } else if (assetTicker === USDT_TICKER) {
       rateMultiplier = prices?.[USDT_COINGECKOID]?.[currency.value] || 1;
-      preferredCurrencyAmount = Big(amountInFiatOrLBTC).times(rateMultiplier);
+      currencyAmount = Big(currencyAmount).times(rateMultiplier);
     } else if (assetTicker === LCAD_TICKER) {
       rateMultiplier = calculateLCAD(prices)[currency.value] || 1;
-      preferredCurrencyAmount = Big(amountInFiatOrLBTC).times(rateMultiplier);
+      currencyAmount = Big(currencyAmount).times(rateMultiplier);
     } else {
       return '';
     }
 
     if (currency.value === 'btc') {
-      return `${preferredCurrencyAmount.toFixed(8)} ${currency.value.toLocaleUpperCase()}`;
-    } else {
-      return `${preferredCurrencyAmount.toFixed(2)} ${currency.value.toLocaleUpperCase()}`;
-    }
-  }, [
-    balanceBaseAmount,
-    currency,
-    lbtcUnit,
-    isLoadingPrices,
-    isErrorPrices,
-    network,
-    prices,
-    selectedMarket,
-  ]);
-
-  const quoteToPreferredCurrency = React.useMemo(() => {
-    // Wait for prices to load (or error out) before displaying conversions
-    if (isLoadingPrices || isErrorPrices || prices === undefined) {
-      return '';
-    }
-
-    let amount = Big(0);
-    try {
-      amount = Big(balanceQuoteAmount);
-    } catch {
-      // ignore user typos, just leave the amount as 0
-    }
-    const assetId = selectedMarket.quoteAsset?.asset_id || '';
-    const assetTicker = selectedMarket.quoteAsset?.ticker || 'unknown';
-    const assetPrecision = selectedMarket.quoteAsset?.precision ?? defaultPrecision;
-
-    let amountInFiatOrLBTC = '';
-    if (isLbtcAssetId(assetId, network)) {
-      amountInFiatOrLBTC = formatLbtcUnitToSats(amount.toNumber(), lbtcUnit);
-      amountInFiatOrLBTC = fromSatsToUnitOrFractional(
-        Number(amountInFiatOrLBTC),
-        assetPrecision,
-        true,
-        'L-BTC'
-      );
+      return `${currencyAmount.toFixed(8)} L-BTC`;
```

Comment:
Why we don't return amount to favorite lbtc unit?

Reply:
I think it looks better to have the base/quote asset converted into L-BTC directly.

It also helps when the base asset is L-BTC (but a different unit) to make sure you are not making a mistake while withdrawing (you get to see the input you are withdrawing in clear L-BTC terms)

But it's your call I can change it.